test/e2e/test_transformer_inference_service.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from kubernetes import client
+
+from kfserving import KFServingClient
+from kfserving import constants
+from kfserving import V1alpha2EndpointSpec
+from kfserving import V1alpha2PredictorSpec
+from kfserving import V1alpha2TransformerSpec
+from kfserving import V1alpha2PyTorchSpec
+from kfserving import V1alpha2CustomSpec
+from kfserving import V1alpha2InferenceServiceSpec
+from kfserving import V1alpha2InferenceService
+from kubernetes.client import V1ResourceRequirements
+from kubernetes.client import V1Container
+from utils import wait_for_isvc_ready
+
+api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION
+KFServing = KFServingClient(config_file="~/.kube/config")
+
+
+def test_transformer():
+    default_endpoint_spec = V1alpha2EndpointSpec(
```

Comment:
why not do this in line?

Reply:
inline makes the spec looks a bit not readable.